.readme-partials/INSTALLATION.md

```diff
@@ -1,13 +1,23 @@
-Clone this repository onto your hard drive and then use Composer to install all dependencies:
+If you normally use WP-CLI on your web host or via Brew, you're most likely using the Phar executable (`wp-cli.phar`). This Phar executable file is the "built", singular version of WP-CLI. It is compiled from a couple dozen repositories in the WP-CLI GitHub organization.
 
-```
-git clone https://github.com/wp-cli/wp-cli-dev
-cd wp-cli-dev
-composer install
-```
-
-This will:
+In order to make code changes to WP-CLI, you'll need to set up this `wp-cli-dev` development environment on your local machine. The setup process will:
 
 1. Clone all relevant packages from the `wp-cli` GitHub organization into the `wp-cli-dev` folder, and
 2. Install all Composer dependencies for a complete `wp-cli-bundle` setup, while symlinking all of the previously cloned packages into the Composer `vendor` folder.
 3. Symlink all folder in `vendor` into corresponding `vendor` folders in each repository, thus making the centralized functionality based on Composer available in each repository subfolder.
+
+First, clone the GitHub repository:
+
+```bash
+git clone https://github.com/wp-cli/wp-cli-dev wp-cli-dev
+```
+
+Before you can proceed further, you'll need to make sure you have [Composer](https://getcomposer.org/), PHP, and a functioning MySQL or MariaDB server on your local machine too.
```

Comment:
Do we want to mention experimental SQLite support here?

Reply:
@swissspidy I'm open to it! What language would you suggest?

Reply:
Maybe something along these lines:

> Alternatively, if you do not have MySQL or MariaDB installed on your computer, you can also run tests using SQLite, which is an alternative database that does not require separate installation. To do so, set the `WP_CLI_TEST_DBTYPE` environment variable accordingly when running tests. Example: `WP_CLI_TEST_DBTYPE=sqlite composer behat`
Note: In this case you do not need to run `composer prepare-tests` either.


Needs some pointer to the Behat docs I suppose.

Reply:
I'll add these in a moment.